ArduPlane/quadplane.cpp

```diff
@@ -328,8 +328,8 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
 
     // @Param: OPTIONS
     // @DisplayName: quadplane options
-    // @Description: This provides a set of additional control options for quadplanes. LevelTransition means that the wings should be held level to within LEVEL_ROLL_LIMIT degrees during transition to fixed wing flight, and the vehicle will not use the vertical lift motors to climb during the transition. If AllowFWTakeoff bit is not set then fixed wing takeoff on quadplanes will instead perform a VTOL takeoff. If AllowFWLand bit is not set then fixed wing land on quadplanes will instead perform a VTOL land. If respect takeoff frame is not set the vehicle will interpret all takeoff waypoints as an altitude above the current position. When Use QRTL is set it will replace QLAND with QRTL for failsafe actions when in VTOL modes. When AIRMODE is set AirMode is automatically enabled if arming by RC channel.
-    // @Bitmask: 0:LevelTransition,1:AllowFWTakeoff,2:AllowFWLand,3:Respect takeoff frame types,4:Use a fixed wing approach for VTOL landings,5:Use QRTL instead of QLAND for failsafe when in VTOL modes,6:Use idle governor in MANUAL,7:QAssist force enabled,8:Tailsitter QAssist motors only,9:enable AirMode if arming by aux switch,10:Enable motor tilt when disarmed,11:Delay spoolup for 2 seconds after arming
+    // @Description: LEVEL TRANSITION:Keep wings within LEVEL_ROLL_LIMIT and only use forward motor(s) for climb during transition, ALLOW FW TAKEOFF: If bit is not set then NAV_TAKEOFF command on quadplanes will instead perform a NAV_VTOL takeoff, ALLOW FW LAND:If bit is not set then NAV_LAND command on quadplanes will instead perform a NAV_VTOL_LAND, VTOL_TAKEOFF_REF: command NAV_VTOL_TAKEOFF altitude is as set by the command's reference frame rather than a delta above current location, USE_FW_APPROACH:Use a fixed wing approach for VTOL landings, USE_QRTL:instead of QLAND for rc failsafe when in VTOL modes, USE_GOVERNOR:Use ICE Idle Governor in MANUAL for forward motor, FORCE_QASSIST: on always,MTRS_ONLY_QASSIST: in tailsitters only, uses VTOL motors and not flying surfaces for QASSIST, AIRMODE_ON_ARM:Airmode enabled when arming by aux switch, DISARMED_YAW_TILT:Enable motor tilt for yaw when disarmed, DELAY_SPOOLUP:Delay VTOL spoolup for 2 seconds after arming, GROUND_EFFECT_COMP: enable ground effect barometer compensation for takeoff and landing
+    // @Bitmask: 0:LEVEL_TRANSITION,1:ALLOW_FW_TAKEOFF,2:ALLOW_FW_LAND,3:VTOL_TAKEOFF_REF,4:USE_FW_APPROACH,5:USE_QRTL,6:USE_GOVERNOR,7:FORCE_QASSIST,8:MTRS_ONLY_QASSIST,9:AIRMODE_ON_ARM,10:DISARMED_YAW_TILT,11:DELAY_SPOOLUP,12:GROUND_EFFECT_COMP
```

Comment:
I'm sorry, but the caps locking of these are far worse, and less readable. `Level transition` is no longer then `LEVEL_TRANSITION` and in fact on most displays will be shorter if you aren't using a mono space font.

Reply:
I'm a bit surprised to see this.  My take-away from the DevCall conversation was that we'd be using e.g. "Level Transition", "Allow FW Takeoff" etc.... with or without caps, followed up by some way of providing an extended description for each bit.